ports/python3/vcpkg.json

```
@@ -1,10 +1,10 @@
 {
   "name": "python3",
   "version-semver": "3.10.0",
-  "port-version": 2,
+  "port-version": 3,
   "description": "The Python programming language",
   "homepage": "https://github.com/python/cpython",
-  "supports": "!(arm | uwp)",
+  "supports": "!uwp & !(windows & arm)",
```

that is probably just to how vpckg tries to build python on windows arm. Is there a definite no from upstream for this configuration?

---

Upstream supports Windows+ARM. When I did the work to build the interpreter, I didn't investigate adding support for Windows ARM beyond a cursory attempt. I can't remember if I was testing the a cursed uwp-arm triplet, though, so it might be worth revisiting.

---

I can think of that it is the same issue as:
https://github.com/microsoft/vcpkg/blob/203daf9844a128537c92b461c3dd7eef18c9c23a/ports/qtbase/cmake/qt_install_submodule.cmake#L82-L84 which is due to the arm toolchain on windows not being correctly setup.

However since it is supported by upstream the support statement here is wrong.
 ```suggestion
  "supports": "!uwp",
```
arm-windows just needs to be added to the ci.baseline to keep ci clean.

---

Hmm... But python3 uses MSBuild. I wish I could remember what the problem was on ARM 😞

---

The only problem is that the python executable is called below to get the extensions, and I've fixed that.
Now support arm on all platforms.